sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryParsingErrors.scala

```diff
@@ -608,8 +608,12 @@ private[sql] object QueryParsingErrors extends QueryErrorsBase {
   }
 
   def unclosedBracketedCommentError(command: String, position: Origin): Throwable = {
-    new ParseException(Some(command), "Unclosed bracketed comment", position, position,
-      Some("_LEGACY_ERROR_TEMP_0055"))
+    new ParseException(
+      Some(command),
+      "Found an unclosed bracketed comment. Please, append */ at the end of the comment.",
+      position,
+      position,
+      Some("UNCLOSED_BRACKETED_COMMENT"))
```

Comment:
Could you replace this by:
```suggestion
    new ParseException(
      command = Some(command),
      start = start,
      stop = stop,
      errorClass = "UNCLOSED_BRACKETED_COMMENT",
      messageParameters = Map.empty)
```
and invoke via:
```scala
      throw QueryParsingErrors.unclosedBracketedCommentError(
        command = command,
        start = Origin(Option(failedToken.getStartIndex)),
        stop = Origin(Option(failedToken.getStopIndex)))
```

Reply:
Cool! Thanks for the suggestion 👍